docs/src/paradox/obtaining-bindgen/docker-container.md

```diff
@@ -1,32 +0,0 @@
-# Using Docker Container
```

Comment:
Will we remove support for bindgen images? (I mean pushing new versions to scalabindgen docker repo)

Reply:
The docker image is less useful now that we publish statically linked executables.

Reply:
... so yeah, we could remove support for the docker images.